src/main/java/org/example/parampammusic/entity/OrderItem.java

```diff
@@ -0,0 +1,37 @@
+package org.example.parampammusic.entity;
+
+import jakarta.persistence.*;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@EqualsAndHashCode
```

Comment:
Please, use same approach for all classes: you should either use annotations or override equals and hashCode.
For constructors the same note!